### DISCUSSION TITLE
Display multiple table in one resource?

### DISCUSSION THREAD

**Discussion by valpuia** [filamentphp/filament#6135] [Help] [answered]
Is there an easy way to display multiple table in list from one resource?

Eg:

Table1 (all detail) (user table)

Table2 (all detail) (settig table)
.....

TableN (all detail) (other table)

All are not related to each other

**Comment by danharrin** [comment, marked as answer]
Using a table widget maybe?

**Reply by valpuia** [reply]
Yes, you are correct.. thanks

**Comment by fikurimax** [comment]
How can I have multiple table within the same livewire page?

Here's what I've tried:
1. create two different method defining the table
2. use `{{ $this->table }}` and `{{ $this->table2 }}`

But I get an error like this `Argument #1 ($table) must be of type Filament\Tables\Table, Filament\Infolists\Infolist given`. The first table using function name `table` runs normally. But the other one gives an error

**Reply by danharrin** [reply]
You may not have more than one table in a Livewire component, as I said above just create a new component / table widget and embed that

**Reply by YashodMahela** [reply]
can we access Relation Manager tables in List Records. relational manager is linked into Edit Record, can these relational managers link to List Records.
